dotcom-rendering/src/web/components/TopRightAdSlot.importable.tsx

```diff
@@ -42,7 +47,7 @@ export const TopRightAdSlot = ({
 		!isServer
 	) {
 		// Show a fixed image asking people to subscribe
```

Comment:
Tiny one - can we remove this comment? It feels out of date now we also show labs articles
```suggestion
```